package.json

```diff
@@ -12,7 +12,7 @@
   "type": "module",
   "scripts": {
     "start": "./bindgen/realm-bindgen.ts",
-    "test": "cd bindgen && mocha",
+    "test": "cd bindgen && tsc --incremental && mocha",
```

Comment:
In general, I'm not a big fan of combined build + test commands as it makes it header to test without building (if you just want to debug by messing around in the build artefacts). Wouldn't it be possible and advantages to instrument Mocha to use `tsx` like [we're doing it elsewhere](https://github.com/realm/realm-js/blob/main/integration-tests/tests/.mocharc.json#L2-L7)?

Reply:
Never mind, I missed the `noEmit` in the `tsconfig.json`: https://github.com/realm/realm-core/blob/ms/bindgen-work/bindgen/tsconfig.json#L7